src/PSR7/Validators/BodyValidator/BodyValidator.php

```diff
@@ -47,12 +49,12 @@ public function validate(OperationAddress $addr, MessageInterface $message) : vo
         }
 
         // does the response contain one of described media types?
-        if (! isset($mediaTypeSpecs[$contentType])) {
+        if (! ($mediaTypeSpec = $this->matchMediaTypeSpec($mediaTypeSpecs, $contentType))) {
```

Comment:
a minor opinion here - let's not assign variables in if statements?

Reply:
Can we just tune CS checks configs for things like this?

Reply:
I'll review how to configure `phpcs`. Maybe it is time to introduce static analyzer which can add extra checks and QA.

Reply:
The doctrine coding standard actually includes this check. Ref: https://github.com/doctrine/coding-standard/blob/master/lib/Doctrine/ruleset.xml see `Forbid assignments in conditions`